simplq/src/styles/adminButtons.module.scss

```diff
@@ -0,0 +1,10 @@
+.main-button-content {
```

Comment:
Could you rename and move this to the buttons scss script? Since it's only used in the stories script, that makes more sense.

Reply:
Done